tools/cbmc/proofs/ARP/ARP_OutputARPRequest_buffer_alloc1/Configurations.json

```diff
@@ -0,0 +1,47 @@
+{
+  "ENTRY": "OutputARPRequest",
+  "CBMCFLAGS":
+  [
+      "--unwind 50",
+      "--unwindset xNetworkBuffersInitialise.0:3,xNetworkBuffersInitialise.1:3,vListInsert.0:3,pxGetNetworkBufferWithDescriptor.0:3,pxGetNetworkBufferWithDescriptor.1:3,vNetworkInterfaceAllocateRAMToBuffers.0:3"
+  ],
+  "OBJS":
+  [
+    "$(ENTRY)_harness.goto",
+    "$(FREERTOS)/lib/FreeRTOS-Plus-TCP/source/FreeRTOS_ARP.goto",
+    "$(FREERTOS)/lib/FreeRTOS-Plus-TCP/source/portable/BufferManagement/BufferAllocation_1.goto",
+    "$(FREERTOS)/lib/FreeRTOS/list.goto",
+    "$(FREERTOS)/lib/FreeRTOS/queue.goto"
+  ],
+  "MINIMUM_PACKET_BYTES": 50,
```

Comment:
This is only being used in one place, right? Why not just write `50` on line 34?